app/models/Classroom.coffee

```diff
@@ -169,7 +171,7 @@ module.exports = class Classroom extends CocoModel
     nextLevel = courseLevels.models[nextIndex]
     nextLevel = arena if levelsLeft is 0
     nextLevel ?= _.find courseLevels.models, (level) -> not levelSessionMap[level.get('original')]?.get('state')?.complete
-    [userStarted, courseComplete, totalComplete] = coursesHelper.hasUserCompletedCourse(userLevels, levelsInCourse)
+    [_userStarted, courseComplete, _totalComplete] = coursesHelper.hasUserCompletedCourse(userLevels, levelsInCourse)
```

Comment:
Why this change?

Reply:
The underscores are to try and show that those variables are unused.